packages/query/src/bitcoin/ordinals/inscriptions.query.ts

```diff
@@ -65,30 +55,17 @@ async function fetchInscriptions({
   return res.data;
 }
 
-export interface GetInscriptionsInfiniteQuery {
-  offset: number;
-  total: number;
-  stopNextFetch: boolean;
-  inscriptions: InscriptionResponseHiro[];
-  fromIndex: number;
-  addressesWithoutOrdinalsNum: number;
-  addressesMap: Record<string, number>;
-}
-
 /**
  * // TO REFACTOR
  * Returns all inscriptions for the user's current account
  */
-export function useGetInscriptionsInfiniteQuery<T extends unknown = GetInscriptionsInfiniteQuery>(
-  {
-    taprootKeychain,
-    nativeSegwitAddress,
-  }: {
-    nativeSegwitAddress: string;
-    taprootKeychain: HDKey | undefined;
-  },
-  options?: UseInfiniteQueryOptions<GetInscriptionsInfiniteQuery, unknown, T>
-) {
+export function useGetInscriptionsInfiniteQuery({
```

Comment:
Do you know if we can do the refactor mentioned above? It could be nice to do it now we are changing this but maybe it's complicated

Reply:
I'm actually not real sure who left that comment to refactor as it was relocated code, so the git lens is off. I would assume it was @alter-eggo bc our use of `useInfiniteQuery` was mostly implemented by him? @alter-eggo can I create an issue for you to relook at these query hooks once I'm done with the Stacks query refactors? These hooks are a bit complicated to understand and think could be refactored? There are several comments left here by @pete-watters with suggestions.

Reply:
Let's merge this anyway and we can consider improvements in the future

Reply:
yeah, it's quite complex logic here as we need to fetch inscriptions from native segwit address, loop through taproot addresses, and also support dynamic loading once reach bottom of home page (if user has many inscriptions)

agree, it would be great to create an issue for it

Reply:
Made issue here: https://github.com/leather-io/issues/issues/109